app/assets/javascripts/components/sampleUpload.jsx

```diff
@@ -778,6 +803,32 @@ class SampleUpload extends React.Component {
                     </div> :
                     null
                 }
+                <div className='field'>
+                  <div className='row'>
+                    <div className='col no-padding s12'>
+                      <div className='field-title'>
+                        <div className='read-count-label'>
+                          Sample name
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <div className='row input-row'>
+                    <div className='col no-padding s12'>
+                      <input type='text' ref='sample_name' className='browser-default' value={this.state.generatedSampleName} onChange={ (e) => {
```

Comment:
I think sample name should be above the admin options.

Reply:
Yeah we didn't want to have the user manually type in a sample name, so we put it at the bottom so we can generate it and have the user edit it

Reply:
but it's below the admin options. users should be able to edit it. 
it's odd to have <user options> <admin options> and then <user options> again.